pkg/utils/etcdutil/etcdutil.go

```diff
@@ -230,3 +232,61 @@ func CreateClients(tlsConfig *tls.Config, acUrls []url.URL) (*clientv3.Client, *
 	log.Info("create etcd v3 client", zap.Strings("endpoints", endpoints))
 	return client, httpClient, nil
 }
+
```

Comment:
Please ignore change in this file. It will merge with https://github.com/tikv/pd/pull/5985 once it's committed.